graf3d/eve/CMakeLists.txt

```diff
@@ -2,36 +2,37 @@
 # CMakeLists.txt file for building ROOT graf3d/eve package
 ############################################################################
 
-include_directories(${OPENGL_INCLUDE_DIR})
+if(geo)
```

Comment:
And else - what's `${headers1}` and `${headers2}`? Can we even build eve without? At least the dicts are going to be empty, right? This looks suspicious :-)

Reply:
Sorry I do not understand the comment. Eve depends on Geom therefore it's all or nothing, right?

Reply:
But then why not prohibit `add_subdirectory(eve)` depending on `if(geo)`?